.codeclimate.yml

```diff
@@ -28,6 +28,7 @@ plugins:
   rubocop:
     enabled: true
     config: ".rubocop_cc.yml"
+    channel: rubocop-0-68
```

Comment:
All the other PRs reference `rubocop-0-69`.
This one is only 68, any reason?

Reply:
0-69 wasn't available at the time.

Reply:
Was updated here btw: https://github.com/ManageIQ/manageiq/pull/18933